common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -53,7 +54,8 @@ case class ActivationMessage(
     activationId: ActivationId,
     activationNamespace: EntityPath,
     content: Option[JsObject],
-    cause: Option[ActivationId] = None)
+    cause: Option[ActivationId] = None,
+    traceMetadata: Option[SpanMetadata] = None)
```

Comment:
@rabbah You had mentioned that there is already a prototype being developed to make logs pluggable. Are you suggesting that Tracing can be made pluggable on top of these mentioned log changes (I am assuming that there would be some core changes added to Openwhisk to support plugins in general )?